notebooks/.ipynb_checkpoints/Practice-checkpoint.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
What is the purpose of this file? Is it necessary to commit this?

Reply:
This file is created automatically but will be added to the `.gitignore`.